components/Card/Card.tsx

```diff
@@ -24,7 +24,7 @@ export const Card: FC<Props> = ({ template, type }) => {
     assetsForSale,
     lowestPrice,
     template_id,
-    collection: { collection_name },
+    collection: { collection_name, name: collectionName },
```

Comment:
Can we rename `collectionName` to `displayName` so it's less confusing?

Reply:
agreed